diff -up netcdf-c-4.4.1.1/h5_test/tst_h_par.c.err netcdf-c-4.4.1.1/h5_test/tst_h_par.c
--- netcdf-c-4.4.1.1/h5_test/tst_h_par.c.err	2016-11-21 11:27:08.000000000 -0700
+++ netcdf-c-4.4.1.1/h5_test/tst_h_par.c	2016-11-29 11:34:59.433806850 -0700
@@ -11,6 +11,7 @@
    $Id: tst_h_par.c,v 1.15 2010/05/25 13:53:04 ed Exp $
 */
 #include <nc_tests.h>
+#include "err_macros.h"
 #include <hdf5.h>
 
 /* Defining USE_MPE causes the MPE trace library to be used (and you
