--- netcdf-4.1-snapshot2009111008/configure.libs	2009-11-10 08:38:04.000000000 -0700
+++ netcdf-4.1-snapshot2009111008/configure	2009-11-11 20:24:32.932280524 -0700
@@ -3420,7 +3420,7 @@
   curl_cflags="-I$with_curl/include"
   curl_dir="$with_curl"
 else
-  curl_cflags=-lcurl
+  curl_cflags=
   curl_libs=
   curl_dir=
 fi
@@ -31317,7 +31317,7 @@
    LDFLAGS="${LDFLAGS} -L${HDF5DIR}/lib"
    CPPFLAGS="${CPPFLAGS} -I${HDF5DIR}/include"
 fi
-LIBS="${LIBS} -lm -lz -lhdf5"
+LIBS="${LIBS} -lz -lhdf5"
 
 # Add the HDF4 directory and library if needed.
 if test "x$enable_hdf4" = xyes; then
@@ -33122,7 +33122,7 @@
    if test "x$ZLIBDIR" != x ; then
       NC_LIBS="$NC_LIBS -L$ZLIBDIR/lib"
    fi
-   NC_LIBS="$NC_LIBS -lz -lm $LIBS"
+   NC_LIBS="$NC_LIBS -lz $LIBS"
    if test "x$SZLIBDIR" != x ; then
       NC_LIBS="$NC_LIBS -L$SZLIBDIR/lib"
       NC_LIBS="$NC_LIBS -lsz $LIBS"
@@ -33138,7 +33138,7 @@
     if test "x$with_daplib" != x ; then
        NC_LIBS="$NC_LIBS -L$with_daplib/lib"
     fi
-    NC_LIBS="$NC_LIBS $CURL_LIBS"
+    NC_LIBS="$NC_LIBS"
 fi
 if test "x$enable_separate_fortran" = xyes ; then
    NC_FLIBS="-lnetcdff $NC_LIBS"
