--- netcdf-4.1-snapshot2009111008/configure.hdf4	2009-11-10 08:38:04.000000000 -0700
+++ netcdf-4.1-snapshot2009111008/configure	2009-11-11 14:51:07.808576404 -0700
@@ -33131,7 +33131,7 @@
       if test "x$HDF4DIR" != x ; then
          NC_LIBS="$NC_LIBS -L$HDF4DIR/lib"
       fi
-      NC_LIBS="$NC_LIBS -lhdf4	"
+      NC_LIBS="$NC_LIBS -lmfhdf -ldf"
    fi
 fi
 if test "x$enable_dap" = xyes ; then
