--- netcdf-4.1-snapshot2009111008/libsrc4/Makefile.in.hdf5	2009-11-10 08:38:09.000000000 -0700
+++ netcdf-4.1-snapshot2009111008/libsrc4/Makefile.in	2009-11-11 16:58:44.846340629 -0700
@@ -791,7 +791,7 @@
 # Our output library will also include contents of the netcdf3
 # convenience library.
 libnetcdf_la_LIBADD = ../libsrc/libnetcdf3.la $(am__append_11) \
-	$(am__append_12) $(am__append_13)
+	$(am__append_12) $(am__append_13) -lhdf5_hl -lhdf5
 @BUILD_V2_TRUE@TST_V2 = tst_v2 
 
 # Netcdf-4 source.
