diff -up netcdf-4.1.2-beta2-snapshot2010120222/nc-config.in.fflags netcdf-4.1.2-beta2-snapshot2010120222/nc-config.in
--- netcdf-4.1.2-beta2-snapshot2010120222/nc-config.in.fflags	2010-12-03 10:03:38.000000000 -0700
+++ netcdf-4.1.2-beta2-snapshot2010120222/nc-config.in	2010-12-03 10:04:26.594466933 -0700
@@ -13,8 +13,8 @@ includedir=${prefix}/include
 cc="@CC@"
 cxx="@CXX@"
 fc="@FC@"
-cflags=" -I${includedir} @EXTERN_CPPFLAGS@" 
-fflags="@FFLAGS@ @MOD_FLAG@${includedir}"
+cflags="-I${includedir} @EXTERN_CPPFLAGS@" 
+fflags="@MOD_FLAG@${includedir}"
 has_dap="@HAS_DAP@"
 has_nc2="@HAS_NC2@"
 has_nc4="@HAS_NC4@"
